corehq/apps/app_manager/models.py

```diff
@@ -1736,6 +1736,8 @@ class SortElement(IndexedSchema):
     direction = StringProperty()
     display = DictProperty()
 
+    def got_display_values(self):
```

Comment:
[nit](https://en.wikipedia.org/wiki/Nitpicking#Metaphor): rename to `has_display_values`